vita3k/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -262,6 +262,9 @@ EXPORT(int, sceAppUtilSaveDataDataSave, SceAppUtilSaveDataFileSlot *slot, SceApp
     TRACY_FUNC(sceAppUtilSaveDataDataSave, slot, files, fileNum, mountPoint, requiredSizeKiB);
     SceUID fd;
 
+    if (requiredSizeKiB)
+        *requiredSizeKiB = 0; // Set this to 0 as its not always initialized to 0 and 0 is expected when all is good (afternoon)
```

Comment:
requiredSizeKiB should be set to the size required for the save file if there is not enough space available.
If there is enough space available, it must be set to 0.
We always assume there is enough space available, so this is the correct behavior, please just update the comment to make it easier to understand, like:
`// requiredSizeKiB must be set to 0 if there is enough space available`

Reply:
Is what i pushed better now?

Reply:
Fine for me, although I don't think the TODO is necessary: I don't see a reason to emulate the fact that there is no more storage space available on the PS Vita.